packages/strapi-bookshelf/lib/index.js

```diff
@@ -40,12 +40,10 @@ module.exports = function(strapi) {
      * Initialize the hook
      */
 
-    initialize: cb => {
+    initialize: async cb => {
       const connections = _.pickBy(strapi.config.connections, { connector: 'strapi-bookshelf' });
 
-      const done = _.after(_.size(connections), cb);
-
-      _.forEach(connections, (connection, connectionName) => {
+      await _.forEach(connections, (connection, connectionName) => {
```

Comment:
I didn't know this syntax... It tested it and it doesn't change anything in my scripts. 

@ayadav Can you explain to me why we need to put the `await` reserved word?

Reply:
@Aurelsicoko You are right, I checked again and it actually had no impact. I have removed those now.